oca_port/port_addon_pr.py

```diff
@@ -54,7 +57,7 @@ def path_to_skip(commit_path):
 class PortAddonPullRequest():
     def __init__(
             self, repo, upstream_org, repo_name,
-            from_branch, to_branch, fork, user_org, addon, storage,
+            from_branch, to_branch, fork, user_org, addon, storage, cache=None,
```

Comment:
storage and cache could be singletons w/o need to pass them around, no?

Reply:
The cache object is created by `utils.cache.UserCacheFactory` (called in `oca_port/__init__.py`) and depends on the `no_cache` CLI parameter. So, either we pass the cache object, or the required parameters, but we'll have to pass something anyway.

However I'll like later to bundle all these parameters in one object, I was thinking about `click.Context`, but I don't want to make exposed classes & functions dependent on `click.Context` to use `oca_port` as a Python package, so maybe a custom context object instead :thinking: From there then yes, it would be possible to build the cache object from anywhere without passing it as parameter.